Example/Tests/GINIImageNoResultsViewControllerTests.swift

```diff
@@ -35,6 +35,7 @@ final class GINIImageNoResultsViewControllerTests: XCTestCase {
     
     override func setUp() {
         _ = viewController.view
+        viewController.captureSuggestions = items
```

Comment:
I have a problem with these tests in general. 

First of all, I don't see a benefit in using the real values used by the app also at runtime.

Secondly, and more important, I don't think it makes sense to have a test for each suggestion. It duplicates tests and the only benefit is that theoretically someone might exchange the indexes. I'd recommend having only one test that tests one of the cells. Maybe not the first one, just pick one.